tests/com/twitter/intellij/pants/execution/PantsOutputMessageTest.java

```diff
@@ -69,4 +74,33 @@ public void testDotFilePathHasNullFilterResult() {
     Filter.Result result = filter.applyFilter(".: message", 10000);
     assertNull("result", result);
   }
+
+  protected static <T> void assertContain(List<? extends T> actual, T... expected) {
```

Comment:
extend from com.intellij.testFramework.UsefulTestCase and use assertContainsElements instead